client/client.js

```diff
@@ -58,38 +68,62 @@ function attachObjectToPlayer(player, boneId, objectName, positionX, positionY,
         natives.requestModel(hashOfProp);
         const modelLoadInterval = alt.setInterval(() => {
             if (natives.hasModelLoaded(hashOfProp)) {
-                alt.clearInterval(modelLoadInterval)
-            } 
+                alt.clearInterval(modelLoadInterval);
+            }
         }, 100);
 
         var newObject = natives.createObject(hashOfProp, player.pos.x, player.pos.y, player.pos.z, true, true, true);
 
         // Release memory for model
         natives.setModelAsNoLongerNeeded(hashOfProp);
-        
-        var boneIndex = natives.getPedBoneIndex(player.scriptID, boneId); 
+
+        var boneIndex = natives.getPedBoneIndex(player.scriptID, Number(boneId));
 
         if (newObject) {
             // Hide weapon before attaching object
             natives.setPedCurrentWeaponVisible(player.scriptID, false, true, true, true);
-
-            natives.attachEntityToEntity(newObject, player.scriptID, boneIndex, positionX, positionY, positionZ, rotationX, rotationY, rotationZ, 
-                false, false, false, false, 1, true);  
+            console.log(boneId, objectName, positionX, positionY, positionZ, rotationX, rotationY, rotationZ);
+            natives.attachEntityToEntity(
```

Comment:
Formatting should also not be changed here and I don't like to numerate the parameters.